pkg/test/framework/features/features.gen.go

```diff
@@ -11,22 +11,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-	
+
 //WARNING: THIS IS AN AUTO-GENERATED FILE, DO NOT EDIT.
```

Comment:
I think this is supposed to come from some yaml source? We love codegen in Istio :slightly_smiling_face: